tests/phpunit/payment-methods/class-wc-stripe-upe-payment-gateway-test.php

```
@@ -1520,6 +1520,62 @@ public function test_checkout_saves_sepa_generated_payment_method_to_order() {
+			++$action_calls;
+		};
+		add_action( 'woocommerce_stripe_add_payment_method', $listener );
+
+		try {
+			$this->mock_gateway->handle_saving_payment_method( $order, $payment_method_object, WC_Stripe_Payment_Methods::IDEAL );
+		} finally {
+			remove_action( 'woocommerce_stripe_add_payment_method', $listener );
+		}
+
+		$this->assertSame( $expected_save ? 1 : 0, $action_calls );
+	}
```

PHPMD (2.15.0)

[error] The method test_handle_saving_payment_method_respects_per_method_toggle is not named in camelCase. (undefined)

(CamelCaseMethodName)